README.md

```diff
@@ -64,76 +66,14 @@ to convert between `Play's` form errors and view models used by `GovukErrorMessa
 ...
 ```
 
-It also provides extension methods on `Play's` [Html](https://www.playframework.com/documentation/2.6.x/api/scala/play/twirl/api/Html.html) objects.
-This includes HTML trims, pads, indents and handling HTML emptiness.
+It also provides extension methods on `Play's` [Html](https://www.playframework.com/documentation/2.6.x/api/scala/play/twirl/api/Html.html) 
+objects. This includes HTML trims, pads, indents and handling HTML emptiness.
 
-### An example usage of [GovukLayout](https://github.com/hmrc/play-frontend-govuk/blob/master/src/main/play-26/twirl/uk/gov/hmrc/govukfrontend/views/layouts/govukLayout.scala.html) template
```

Comment:
Removing this section as it's out of date, encourages inline scripts and is a duplication of the content on https://github.com/hmrc/play-frontend-govuk/blob/master/docs/maintainers/standalone.md